### DISCUSSION TITLE
Recreational landings

### DISCUSSION THREAD

**Discussion by aliwhitman** [shcaba/Sebastes_melanops_OR#10] [General]
Just documenting some of the decisions we discussed at yesterday's meeting.  Jason, let me know if I got any of these incorrect! 

With Ocean boat catches, we are going with the most recent sport reconstruction for historical landings (1979 - 2000) and use a ramp from 1973 - 1978. 

For Shore/estuary landings, we are going with Ali's current reconstruction as she uses the filtered weights from MRFSS to reconstruct biomass. Noting the 10 year average of landings from 1996 - 2005 is used to estimate recent landings (2006 - 2022) rather than the previous assessment's linear relationship with fish licenses. We are also employing a ramp from 1892 - 1979 to account for landings prior to the availability of MRFSS data. 

For the final landings time series, we will need to add an estimate of discarded mortality.  Reviewing the available data from RecFIN (2001 - 2022), the average % discarded is 0.92%.  Table is below.  If there are no objections, I will add this to the two recreational fleets individual time series.  

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/daubleal/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/daubleal/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Year | Sum of RETAINED_KG | Sum of RELEASED_DEAD_KG | Percent of total
-- | -- | -- | --
2001 | 289705.13 | 1015.04 | 0.35%
2002 | 284070.56 | 542.90 | 0.19%
2003 | 320539.03 | 638.81 | 0.20%
2004 | 323630.49 | 642.90 | 0.20%
2005 | 301860.04 | 2014.63 | 0.66%
2006 | 257728.30 | 1378.79 | 0.53%
2007 | 244359.74 | 3066.77 | 1.24%
2008 | 223662.08 | 4005.63 | 1.76%
2009 | 274852.26 | 2763.02 | 1.00%
2010 | 289669.32 | 2473.07 | 0.85%
2011 | 196975.64 | 1739.93 | 0.88%
2012 | 208914.85 | 1221.13 | 0.58%
2013 | 319524.59 | 1781.07 | 0.55%
2014 | 340313.11 | 2234.56 | 0.65%
2015 | 453818.70 | 4787.79 | 1.04%
2016 | 400910.18 | 3742.96 | 0.92%
2017 | 396778.85 | 5893.89 | 1.46%
2018 | 275091.69 | 3687.50 | 1.32%
2019 | 301136.06 | 3937.15 | 1.29%
2020 | 315644.07 | 4808.63 | 1.50%
2021 | 316974.53 | 3755.61 | 1.17%
2022 | 367429.05 | 6834.03 | 1.83%
  |   | Mean: | 0.92%

</body>

</html>

**Comment by shcaba** [comment]
Yes to all of the above. Thank you.

**Comment by shcaba** [comment]
Would we have those this week? Just wondering what I will be able to present at the pre-assessment workshop.

**Reply by aliwhitman** [reply]
Yes, I want to have these done today or tomorrow. 

One quick note - I just realized that typically I don't recommend adding discards to recreational time series prior to 2000 ("historic" time period for recreational data).  Recreational bag limits prior to the overfished designations in the early 2000s were very liberal, so regulatory discards would be minimal to non-existent.  Some folks might throw back small fish but there are also no minimum size limits either.  

So that would be a relatively small change in how we applied discards from the last assessment, given the low discard rate. Let me know if you're okay with discards only 2001 forward for both recreational time series or if you'd like to discuss further.

**Comment by shcaba** [comment]
Yes, that is consistent with what you mentioned to the canary team this morning, and I am totally fine with that.

**Comment by shcaba** [comment]
I have assembled text and the figures you provided on boat and shore-based catches (see GitHub page). Do those include the discard rates?

**Reply by shcaba** [reply]
And please add more text or correct anything as needed.

**Reply by aliwhitman** [reply]
The recreational time series provided in the google drive does include discards during the 2001 - 2022 time period.  I will take a look at the page for the workshop.
